app/services/location_searcher.rb

```diff
@@ -6,10 +6,11 @@ def initialize(search_form)
     @address_coordinates = Geocoder.coordinates search_form.address
     @search_radius = search_form.search_radius
     @specialties = search_form.specialties
+    @confirmed_by_gi = search_form.confirmed_by_gi
   end
 
   def search
-    locations = Location.near(@address_coordinates, @search_radius).to_a
+    locations = Location.confirmed_by_gi(@confirmed_by_gi).near(@address_coordinates, @search_radius).to_a
```

Comment:
If the scope is updated in `Location`, you'll need to rewrite this to conditionally apply the scope to the results